data/services.json

```diff
@@ -1,4 +1,35 @@
 [
+  {
+    "target": [
+      "researchers"
+    ],
+    "type": [
+      "Tool"
+    ],
+    "search_tags": [ 
+      "GeneSPIDER tutorial",
+      "Gene regulatory network inference",
+      "Benchmarking tools",
+      "MATLAB",
+      "Git",
+      "Bitbucket",
+      "Simulation",
+      "Experiment design",
+      "Installation guide",
+      "Tool",
+      "SciLifeLab",
+      "Data Centre"
+    ],
+    "name": "GeneSPIDER version: v2",
+    "description": "GeneSPIDER(v2) is a Matlab toolbox for gene regulatory network inference and benchmarking with controlled network and data properties.",
+    "thumbnail": "/img/service_thumbnails/GeneSPIDER.png",
+    "thumbnail_border": true,
+    "url": "https://bitbucket.org/sonnhammergrni/genespider/src/master/",
```

Comment:
Shouldn't this be https://sonnhammer-tutorials.bitbucket.io/genespider.html#Getting_started i.e. the link to application rather then the repository ?

Reply:
Yes, I think that this change can be made (to un-stall the PR)